Longest_Palindromic_Substring/Longest_Palindromic_Substring.java

```diff
@@ -0,0 +1,80 @@
+/*
+    LONGEST PALINDROMIC SUBSTRING
+
+    The task is to find the longest substring of a given string that is palindromic.
+    A palindromic string is a string which is same when read from front or back.
+
+    eg. radar is same when read frontwards or backwards.
+
+*/
+
+import java.io.*;
```

Comment:
Please avoid using * in import statements

Reply:
Sure I'll remove that. Anything else ?

Reply:
Still reviewing, I will add all the comments by tomorrow